6tempcontrol

```diff
@@ -214,6 +221,12 @@ do
   GPU=0
   while (( GPU < GPUS ))
   do
+    if [ ${DISABLED_GPU_ARRAY[$GPU]} ]; then
+      # skip if disabled
+      echo "${B}GPU $GPU -${N} ${R}Disabled"
```

Comment:
Line 226 should be changed to:
echo -e "${B}GPU $GPU,${N} ${R}${B}Disabled${N}"